### ISSUE TITLE
New make/library question

### ISSUE THREAD

**Issue by matthewholding28** [Gaius-Augustus/Augustus#135]
Hello, and thank you for your time. 

I have seen a number of issues with compiling augustus from source on this thread, and understand that there are issues with installing the current version, particularly on remote clusters. 

Unfortunately, my specific issue with a missing library file doesn't appear to be covered here yet. 

I am compiling the latest Augustus from source cloned from github, and I am working on my university's HPC on the following release: centos-release-7-6.1810.2.el7.centos.x86_64

Here is the 'make' compile error below. I tried I would greatly appreciate your advice in compiling the software:

make[1]: Entering directory `/data/gpfs/home/user/software/Augustus/src'
g++ -c -Wall -Wno-sign-compare -pedantic -g -ggdb -O3  -o randseqaccess.o randseqaccess.cc -I../include -I/usr/include/mysql -I/cm/local/apps/mysql++/current/ 
In file included from randseqaccess.cc:10:0:
../include/randseqaccess.hh:20:21: fatal error: mysql++.h: No such file or directory
 #include <mysql++.h>

Best,
Matt

**Comment by MarioStanke** [comment]
You are missing the MySQL library.

In README.md
```## Install dependencies

The following dependencies are required for AUGUSTUS:
  - libsqlite3-dev
  - libmysql++-dev
...
```

If you can't install these libraries and don't need comparative augustus (which annotates multiple genomes simultaenously), you can alternatively comment out ```COMPGENEPRED = true``` in ```common.mk```.

**Comment by matthewholding28** [comment]
Dear Mario, 

Thank you for your reply. 

Update: The major issue turned out to be attempting to run augustus (and braker) on a CentOS system when the libraries called were specific to ubuntu. I ended up solving this by creating an Amazon Web Service Ubuntu 18.02  instance and installing everything into a singularity sandbox, which I then moved over to my HPC for execution. For others that have these types of compatibility issues, I will say that this was a relatively painless process, particularly since I could easily apt-get install all of the needed library dependencies.
